benchmarks/perfetto/backends/clp.py

```diff
@@ -0,0 +1,10 @@
+import argparse
+from .common import TraceAnalysis
+
+class CLPTraceAnalysis(TraceAnalysis):
+
+    def __init__(self, args: argparse.Namespace):
+        super().__init__(args)
+
+    def load(self, input):
+        raise NotImplementedError("CLP load is not implemented.")
```

Comment:
Do we need this since the default implementation of `load` already prints a similar error?

Reply:
Thanks, fixed